sources/packages/backend/apps/api/src/route-controllers/student/student.controller.service.ts

```diff
@@ -154,9 +154,11 @@ export class StudentControllerService {
       );
     }
 
+    const encodedFileName = encodeURIComponent(studentFile.fileName);
```

Comment:
I think we need to investigate it better. I saved a file with double quotes, comma and question mark. 
![image](https://github.com/user-attachments/assets/2193aa38-083a-4cf2-b1a7-cf3ebd65b938)

I couldn't event download that back.
![image](https://github.com/user-attachments/assets/59da33dc-c7cf-4578-98b5-7758344ad4b8)

The name saved seems to be the issue:
![image](https://github.com/user-attachments/assets/83b36d2f-1006-4407-8a56-a6c19d267aba)

Reply:
Good! That's interesting it didn't work for me in my local env... Thanks for checking that!